src/include/mempool.h

```diff
@@ -240,9 +263,16 @@ const char *get_pool_name(pool_index_t ix);
 struct type_t {
   const char *type_name;
   size_t item_size;
-  ceph::atomic<ssize_t> items = {0};  // signed
+  struct type_shard_t {
+    ceph::atomic<ssize_t> items = {0}; // signed
+    char __padding[128 - sizeof(ceph::atomic<ssize_t>)];
+  } __attribute__ ((aligned (128)));
+  type_shard_t shards[num_shards];
```

Comment:
The counter sharding is not crimson-specific but has a global effect. Personally I think it's the right thing but I would really prefer to avoid self-approving.

Reply:
should be addressed by https://github.com/ceph/ceph/pull/53130/commits/6e2060bf0f80f1af637092f7ca0c39628bb4215e